fuzzers/072-ordered_wires/run_fuzzer.py

```diff
@@ -0,0 +1,49 @@
+import threading
+import os
+import shutil
+
+def start_vivado(blockID, start, stop):
+    print ("Running instance :" + str(blockID))
+    os.system("${XRAY_VIVADO} -mode batch -source $FUZDIR/job.tcl -tclargs " + str(blockID) + " " + str(start) + " " + str(stop))
+
```

Comment:
Use subprocess.check_call

Reply:
I'll take a look in about 20 hours. Sorry I'm a bit rusty in python.